singleton.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+
+
+class Singleton(type):
+    def __init__(cls, name, bases, attrs, **kwargs):
+        super().__init__(name, bases, attrs)
```

Comment:
super() calls should look like super(Singleton, cls). We have to specify who's super do we want to call. Also we either remove the **kwargs argument from __init__'s definition or we pass it to super __init__ call.